config/staging/iatlas-db.yaml

```diff
@@ -9,9 +9,9 @@ stack_tags:
   OwnerEmail: "andrew.lamb@sagebase.org"
 parameters:
   VpcName: iatlasvpc
-  KmsKeyName: "/alias/iatlas-kms-staging/KmsKey"
+  KmsKeyName: !stack_output_external iatlas-kms-staging::KmsKeyArn
```

Comment:
KmsKeyName eventually gets assigned to `PerformanceInsightsKMSKeyId` in templates/aurora-postgresql-db.yaml.  So what you want to pass in is the Key ID not the name nor the ARN